nixos/tests/firefly-iii-data-importer.nix

```diff
@@ -0,0 +1,27 @@
+import ./make-test-python.nix (
+  { lib, ... }:
+
+  {
+    name = "firefly-iii-data-importer";
+    meta.maintainers = [ lib.maintainers.savyajha ];
+
+    nodes.dataImporter =
+      { ... }:
+      {
+        services.firefly-iii-data-importer = {
+          enable = true;
+          enableNginx = true;
+          settings = {
+            LOG_CHANNEL = "stdout";
+            USE_CACHE = true;
+          };
+        };
+      };
+
+    testScript = ''
+      dataImporter.wait_for_unit("phpfpm-firefly-iii-data-importer.service")
+      dataImporter.wait_for_unit("nginx.service")
+      dataImporter.succeed("curl -fvvv -Ls http://localhost/token | grep 'Firefly III Data Import Tool'")
```

Comment:
Ideally this would test if it could import a csv, but as this would be annoying to setup, I would not block merging this PR.
Perhaps add a TODO to test actual importing?

Reply:
We'd have to generate a CSV during startup, I think? I'm not sure. If it's sending the correct html over it should be functional, but I'm a little wary of replicating CI tests here.